azurerm/resource_arm_iothub.go

```diff
@@ -332,3 +564,148 @@ func flattenIoTHubSharedAccessPolicy(input *[]devices.SharedAccessSignatureAutho
 
 	return results
 }
+
+func flattenIoTHubEndpoint(input *devices.RoutingProperties) []interface{} {
+	results := make([]interface{}, 0)
+
+	if input != nil && input.Endpoints != nil {
+
+		for _, container := range *input.Endpoints.StorageContainers {
```

Comment:
can we nil-check around this to ensure `input.Endpoints.StorageContainers` isn't nil?